api/pom.xml

```diff
@@ -37,6 +37,16 @@
 			<artifactId>openmrs-test</artifactId>
 			<type>pom</type>
 		</dependency>
+
+		<dependency>
+			<groupId>org.openmrs.module</groupId>
+			<artifactId>metadatadeploy-api</artifactId>
+		</dependency>
+
+		<dependency>
+			<groupId>org.openmrs.module</groupId>
+			<artifactId>metadatasharing-api</artifactId>
+		</dependency>

```

Comment:
Do we need to depend on metadata sharing? Isn't it just Metadata deploy we need?

Reply:
@mogoodrich metadata sharing is needed in the application context for running tests

Reply:
Did you then give it a scope of test?

Reply:
@dkayiwa Thank you for that pointer. I have updated the dependency to a test cope and removed it from the config.xml

Reply:
got it, thanks!